lib/LaTeXML/Package/xparse.sty.ltxml

```diff
@@ -15,6 +15,14 @@ use strict;
 use warnings;
 use LaTeXML::Package;
 
+# Newer versions of xparse *may* want to defer to one of these versioned files
+# They are also acceptable for interpretation, but must avoid the de-versioning hack/fallback.
+AssignMapping('INTERPRETABLE_SOURCES', 'xparse-2018-04-12.sty' => 1);
+AssignMapping('INTERPRETABLE_SOURCES', 'xparse-2020-10-01.sty' => 1);
```

Comment:
To be ridiculously pedantic, if you call `\usepackage{xparse-2018-04-12}` directly, you end up reading `xparse.sty` anyway. It's ok because of how `xparse.sty` is written, and clearly you are never supposed to call the versioned xparse directly. But I wonder if it is the right mechanism in general (I don't know how versioned packages are supposed to work in texlive).

Reply:
Indeed, but explicitly pre-loading `xparse-2018-04-12` *only* works because of the way  `xparse` is written. What I like about @dginev's approach is that it doesn't rely on xparse internals, or even whether `xparse-2018-04-12` *is* a "versioned" file (as opposed to simply a file with a funny name). And thus the technique will be more easily adapted to other such situations.

Ultimately, this whole issue is trying to work around what has to be called a hack (but a *very* useful hack) to prefer curated latexml-specific bindings over raw style files and not be mislead by the randomly "versioned" files typically found on arXiv. The hack tends to assume that files with names like "foo-<somedate>.sty" are versioned, that is a snapshot or slightly tweaked version of "foo.sty". But sometimes a file is just a file.